edge_computing/image-base-install/ibi-understanding-image-based-install.adoc

```diff
@@ -0,0 +1,73 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="ibi-image-based-install_{context}"]
+= Understanding an image-based installation for {sno} clusters
+include::_attributes/common-attributes.adoc[]
+:context: ibi-understanding-image-based-install
+
+toc::[]
+
+Image-based installations streamline the deployment process for {sno} clusters, reducing both installation and deployment times. By leveraging an image-based approach, you can preinstall configured and validated instances of {sno} on target hosts. These prepared hosts can be rapidly deployed at the far edge of the network with minimal intervention by using zero touch configuration (ZTC).
+
+== Understanding image-based installation and deployment for {sno} clusters
+
+Service providers continue to deploy their infrastructure at the far edge of the network. However, this presents challenges in terms of the cost, time, and expertise needed for installing and deploying {sno} clusters at remote sites on an enterprise scale. 
+
+At the far edge of the network, challenges such as low bandwidth and high latency can increase installation times and complicate {sno} cluster deployments. These circumstances require experienced technicans to spend considerable time at remote sites, overseeing the installation and deployment of a {sno} cluster.
+
+An image-based installation and deployment overcomes the deficiencies of this workflow by separating the installation and deployment stages for {sno} clusters. Using an image-based installation, you can preinstall multiple hosts with {sno} at a central site, such as a service depot or a factory.
+You can also validate the base configuration for these hosts and leverage the image-based approach to perform reproducible preinstalls at scale using a single container image.
+The preinstalled and validated hosts are shipped to a remote site where they can be rapidly deployed, with minimal technical knowledge, using ZTC.
+
+=== Image-based installation for {sno} clusters
+Using the {lcao}, you can generate a container image that encapsulates an instance of a {sno} cluster. This image is derived from a dedicated cluster that you can configure with target {product-title} version, Day 2 Operators, and configurations.  
+
+You can use this image to consistently preinstall configured and validated instances of {sno} to multiple hosts. The image is an OCI container image that you can install to a target host as a new `ostree` stateroot. This approach enables the preparation of hosts at a central location, for example in a factory or service depot, before shipping the preinstalled hosts to a remote site for rapid deployment using ZTC. 
+
+=== Zero touch configuration for {sno} clusters
```

Comment:
I think it will help if this this section get some context similar to what we have here:
https://docs.openshift.com/container-platform/4.9/scalability_and_performance/ztp-deploying-disconnected.html#about-ztp-and-distributed-units-on-single-node-clusters_ztp-deploying-disconnected

Reply:
I added some further context regarding hub-spoke architecture and how ZTC plays into this... let me know if that works or you had something else in mind.